content/en/real_user_monitoring/_index.md

```diff
@@ -126,8 +126,26 @@ All Datadog SDKs traffic is transmitted over SSL (default 443) to the following
 
 Access RUM by navigating to [**Digital Experience > Performance Summary**][1].
 
+Select an application from the top navigation, or follow the setup instructions for [browser][15] or [mobile][16] to add your first application.
+
+{{< img src="real_user_monitoring/rum-performance-application-selector.png" alt="Select a RUM application" >}}
+
 **Tip**: To open RUM from Datadog's global search, press <kbd>Cmd</kbd>/<kbd>Ctrl</kbd> + <kbd>K</kbd> and search for `real user monitoring`.
 
+## Performance monitoring summary
```

Comment:
I like the content, but I feel we're again missing something that shows what the mobile version looks like. Could we put two screenshots side by side, one for web, one for mobile, please?

Note: to take a good screenshot of a mobile app for the Demo org, you might want to deselect `user` from the Session Type filter as almost all of the traffic actually comes from Synthetics.

Reply:
Done - thanks for the suggestion!